tests/integration/integration-tests.yml

```diff
@@ -4079,6 +4079,54 @@
           - test_five['circuit_termination']['pp_info'] == "PP10-24"
           - test_five['circuit_termination']['description'] == "Test description"
           - test_five['msg'] == "circuit_termination test_circuit_a deleted"
+          -
+
+##
+##
+### NETBOX_SERVICE
+##
+##
+
+    - name: "1 - Device with required information needs to add new service"
```

Comment:
Can you add two more tests? One to test idempotence, just try create the first one, but then create an identical test and make sure it reports that it already exists. And the other one is updating the service, make sure that is tested.

You can also just use an existing device that gets created within `tests/integration/netbox-deploy.py` for this as well.